ddtrace/contrib/pytest/_plugin_v2.py

```diff
@@ -38,6 +50,69 @@
 _NODEID_REGEX = re.compile("^((?P<module>.*)/(?P<suite>[^/]*?))::(?P<name>.*?)$")
 
 
+def _handle_itr_should_skip(item, test_id) -> bool:
+    """Checks whether a test should be skipped, and returns a code collector if needed.
```

Comment:
@romainkomorndatadog I did not catch what is meant by "returns a code collector" here, as it always returns a boolean.

Reply:
That's because I'm terrible at updating comments when I change code. :(